WordPress/Classes/Networking/AccountServiceRemoteREST.m

```diff
@@ -80,14 +80,16 @@ - (RemoteUser *)remoteUserFromDictionary:(NSDictionary *)dictionary
 
 - (NSArray *)remoteBlogsFromJSONArray:(NSArray *)jsonBlogs
 {
-    NSMutableArray *remoteBlogs = [NSMutableArray arrayWithCapacity:[jsonBlogs count]];
-    for (NSDictionary *jsonBlog in jsonBlogs) {
-        BOOL isJetpack = [jsonBlog[@"jetpack"] boolValue];
-        if (!isJetpack || JetpackREST.enabled) {
-            [remoteBlogs addObject:[self remoteBlogFromJSONDictionary:jsonBlog]];
-        }
+    NSArray *blogs = jsonBlogs;
+    if (!JetpackREST.enabled) {
```

Comment:
This logic is slightly different from what we originally had: `jetpack` blocks were being filtered, no matter what the `JetpackREST.enabled` value was. (While the new filtering snippet only checks on the latest to run the filter).

Is that what we wanted?